tokens/tsm-report.go

```diff
@@ -0,0 +1,37 @@
+package tokens
+
+import (
+	"encoding/json"
+
+	"github.com/fxamacker/cbor/v2"
+)
+
+const TSMReportMediaType = "application/vnd.veraison.tsm-report+cbor"
+
+type TSMReport struct {
+	AuxBlob       []byte           `json:"auxblob,omitempty"`
+	OutBlob       []byte           `json:"outblob"`
```

Comment:
By default, the JSON marshaler will encode `[]byte` as base64 with padding.

Instead, we want to model:
```
binary-string = base64url-string .feature "json" / bstr .feature "cbor"

base64url-string = tstr .b64u bstr
```
which is base64 URL-encoding without padding.

I suggest adding a `BinaryString` type and its corresponding JSON marshaler:

```go
// BinaryString is base64url (§5 of RFC4648) without padding.
type BinaryString []byte

func (o BinaryString) MarshalJSON() ([]byte, error) {
	return json.Marshal(
		base64.RawURLEncoding.EncodeToString(o),
	)
}
```

and then use it instead of `[]byte`:

```suggestion
	AuxBlob         BinaryString `json:"auxblob,omitempty"`
	OutBlob         BinaryString `json:"outblob"`
```